### ISSUE TITLE
Bug Report: Play as Pico Message Bug

### ISSUE THREAD

**Issue by NULLSonic** [FunkinCrew/Funkin#3263]
### Issue Checklist

- [X] I have properly named the issue
- [X] I have checked the issues/discussions pages to see if the issue has been previously reported

### What platform are you using?

Itch.io (Downloadable Build) - Windows

### If you are playing on a browser, which one are you using?

None

### Version

0.5.0

### Context (Provide images, videos, etc.)

Upon completing any week you are always greeted with the "You can now play as Pico" message.
[Video showcasing the bug](https://streamable.com/dasm7n)

### Steps to reproduce (or crash logs, errors, etc.)

You can just finish any week and the message should appear.

**Comment by AbnormalPoof** [comment]
duplicate of #3165

**Comment by CyndaquilDAC** [comment]
> duplicate of #3165

No, it isn't. I mean, it partially is (the fact that it pops up every time), but it shouldn't be popping up at all after any week besides weekend 1.

**Comment by AbnormalPoof** [comment]
Misread oops, the brain is not braining today

**Comment by EliteMasterEric** [comment]
Current behavior is that the game nags you whenever you complete a story week while you have a new character to view in Freeplay, and it keeps nagging you until you properly unlock the character.

We may want to change this behavior in the future.

**Comment by CyndaquilDAC** [comment]
> Current behavior is that the game nags you whenever you complete a story week while you have a new character to view in Freeplay, and it keeps nagging you until you properly unlock the character.
> 
> We may want to change this behavior in the future.

Probably a good idea. i guess it makes sense for this update since its introducing playable characters for the first time, but in the future it'll be more understood and this will probably get annoying to people

**Comment by Lasercar** [comment]
Old bug?

**Comment by Hundrec** [comment]
Still happens in 0.6.2

**Comment by Raltyro** [comment]
god
